### PR TITLE
[NETBEANS-6438] Add code folding for PHP attributes

### PR DESCRIPTION
**Pull request by junichi11**
- https://issues.apache.org/jira/browse/NETBEANS-6438
- Add code folding for attributes

##### Expand

nb-6438-folding-attributes

##### Collapse
nb-6438-folding-attributes-collapse

### CODE REVIEW

**Comment by junichi11** [comment]
@tmysik Could you please have a look at this? If there is no problem, let's merge it. Thanks!

File: php/php.editor/src/org/netbeans/modules/php/editor/csl/PHPFoldingProvider.java [hunk 1]
```diff
@@ -31,7 +31,8 @@
  */
 @MimeRegistration(mimeType = FileUtils.PHP_MIME_TYPE, service = FoldTypeProvider.class, position = 1000)
 public class PHPFoldingProvider implements FoldTypeProvider {
-    private static final Collection<FoldType> TYPES = new ArrayList<>(8);
+
+    private static final Collection<FoldType> TYPES = new ArrayList<>(9);
```
Comment (tmysik) [inline php/php.editor/src/org/netbeans/modules/php/editor/csl/PHPFoldingProvider.java:35]: Perhaps `Arrays.asList()`?

**Review by tmysik — APPROVED** [review]

File: php/php.editor/src/org/netbeans/modules/php/editor/csl/PHPFoldingProvider.java (hunk 1 above)
Comment (tmysik) [reply php/php.editor/src/org/netbeans/modules/php/editor/csl/PHPFoldingProvider.java:35]: Just a nitpick

Comment (junichi11) [reply php/php.editor/src/org/netbeans/modules/php/editor/csl/PHPFoldingProvider.java:35]: I also think so :) I'll fix it next time. Thanks!

**Comment by junichi11** [comment]
@tmysik Thank you for your review!

**Pull request merged**